src/datasources/account/account.datasource.spec.ts

```diff
@@ -13,19 +13,37 @@ import {
 import { accountBuilder } from '@/domain/account/entities/__tests__/account.builder';
 import { verificationCodeBuilder } from '@/domain/account/entities/__tests__/verification-code.builder';
 import { getAddress } from 'viem';
+import * as fs from 'fs';
+import * as path from 'path';
```

Comment:
Nitpick: it might be better to select the imported functions instead of the whole modules:
```javascript
import { readFileSync } from 'fs';
import { join} from 'path';
```

Reply:
Fixed in [765de50](https://github.com/safe-global/safe-client-gateway/pull/1235/commits/765de5047d9dced2c9b007dc2c015a71ba9a4423)